presentation/src/main/res/layout/activity_create_category_flow.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.screens.records.create_record.CreateRecordActivity"
+    android:background="@color/brightBackground"
+    >
+
+
+
+    <RelativeLayout
+        android:id="@+id/headView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/closeBt"
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:layout_alignParentRight="true"
+            android:layout_alignParentTop="true"
+            android:layout_marginRight="16dp"
+            android:layout_marginTop="10dp"
+            android:src="@drawable/ic_close_black_24dp"
+            android:tint="@color/gray_subtitle"
+            android:scaleType="center"
+            />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Nieuwe Eigenschappen"
+            android:layout_marginTop="16dp"
+            android:layout_marginLeft="16dp"
+            />
+
+        <io.forus.me.android.presentation.view.component.text.TextView
+            android:id="@+id/title"
+            style="@style/TopHeader"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Choose Type"
+            android:layout_centerHorizontal="true"
+            android:layout_marginTop="40dp"
+            android:paddingEnd="27dp"/>
+
+        <TextView
+            android:id="@+id/description"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="8dp"
+            android:layout_below="@+id/title"
+            android:layout_marginLeft="22dp"
+            android:layout_marginRight="22dp"
+            android:layout_centerHorizontal="true"
+            android:layout_marginTop="6dp"
+            android:gravity="center"
+            android:paddingEnd="27dp"
+            android:visibility="visible"
+            android:padding="8dp"
+            android:text="Select the category under which the new record type should be added"
```

Comment:
Could you add this to strings? Dutch translation is 'Wat voor type eigenschap wilt u aanmaken?'

Reply:
I think the english text is also incorrect. A type is not a category @jknnr

Reply:
Select the type of a new record